src/test/java/com/github/redouane59/twitter/nrt/ITwitterClientV2Test.java

```diff
@@ -37,6 +40,30 @@
   @BeforeAll
   public static void init() {
     twitterClient = new TwitterClient();
+    twitterClient.setAPIListener(new IAPIEventListener(){
```

Comment:
Does it mean that in order to use the stream feature, user has to instantiate all that?